layer/layer_store.go

```diff
@@ -790,6 +804,32 @@ func (ls *layerStore) DriverName() string {
 	return ls.driver.String()
 }
 
+// pruneReport keeps information about deleted data as a result of calling Prune() on a Store.
+type pruneReport struct {
+	RemovedDriverLayers []string
+}
+
+func (ls *layerStore) prune(txData []fileMetadataTxData) *pruneReport {
```

Comment:
nit: this doesn't need to return a pointer. The struct only contains a single pointer itself so we might as well copy it for now. Alternatively, unless we think this report will grow later, is there any reason to return the wrapper struct? Maybe just return the slice directly.

Reply:
I started with a struct being inspired by a prune report structure, but yes, I agree there is no much sense in having a struct here. It's an internal type anyway, we'll easily change it in the future if we need to extend the report.

Reply:
Simplified